Wrapping/Generators/Python/Tests/test_xarray.py

```diff
@@ -124,7 +124,7 @@
     data_array = itk.xarray_from_image(image, view=True)
     assert data_array.name == image.GetObjectName()
     # verify we can run a computation on xarray data
-    assert float(data_array.min().compute()) == 1.0
+    assert float(data_array.min().compute()) > -1.0
```

Comment:
Would the following still be OK?

    assert 0.0 <= float(data_array.min().compute()) <= 255.0 

Just curious... I just learned that Python allows an `a <= x <= b` syntax to denote "in-between" 😃

Reply:
This seems wrong to me.  Why convert the integer to a float and then use integer based comparision?

```python
assert float(data_array.min().compute()) => 0.0

or 

assert int(data_array.min().compute()) => -1
```

Reply:
@hjmjohnson @N-Dekker will add both suggestions, thanks